### DISCUSSION TITLE
How to cancel a single product of a multiproduct order?

### DISCUSSION THREAD

**Discussion by holgerm** [pretix/pretix#4318] [Q&A]
How can a customer (or an admin) cancel a single product of an order that contains multiple products. In my case I (as admin) see only the option to cancel the complete order. I did not find an option to enable such a partly cancelation. Do I miss something or is this really not a feature yet?

This is laborious in particular when the other products ordered are already completely sold and therefore must be savely reassigned to that customer etc.

**Comment by raphaelm** [comment]
It's not implemented for customers. As an admin, click "change products" within an order and you can remove positions.

**Comment by AlexanderWillner** [comment]
There is PR https://github.com/pretix/pretix/pull/5672 related to this question.
